data/avro-schema/src/main/resources/avro/net/corda/data/membership/rpc/certificates/response/CertificateImportedRpcResponse.avsc

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "record",
+  "name": "CertificateImportedRpcResponse",
```

Comment:
If this was `ImportCertificateRpcResponse` the body could have a flag to indicate whether the operation was successful or not. What happens when persistence fails in the currrent case?